Plugins/Events/Events/ObjectEvents.cpp

```diff
@@ -10,11 +11,15 @@ using namespace NWNXLib::API::Constants;
 static NWNXLib::Hooks::Hook s_AddLockObjectActionHook;
 static NWNXLib::Hooks::Hook s_AddUnlockObjectActionHook;
 static NWNXLib::Hooks::Hook s_AddUseObjectActionHook;
+static NWNXLib::Hooks::Hook s_AddOpenObjectActionHook;
```

Comment:
Please rename the hook variables and the hook functions to `s_<OriginalFunctionName>Hook` and `<OriginalFunctionName>Hook` so: `s_OpenInventoryHook`/`s_CloseInventoryHook` and `OpenInventoryHook`/`CloseInventoryHook`